figures/views.py

```diff
@@ -349,11 +349,10 @@ def get_queryset(self):
         queryset = figures.sites.get_users_for_site(site)
         return queryset
 
-    def retrieve(self, request, pk, *args, **kwargs):
```

Comment:
With adding `get_serializer_context` along with site filtering in the `get_queryset` method, we no longer need to override `retrieve` and we don't need to add an overriding `list` method either